src/main/java/com/skynest/uitesting/tests/UserRegisterTest.java

```diff
@@ -0,0 +1,21 @@
+package com.skynest.uitesting.tests;
+
+import org.testng.annotations.Test;
+
+public class UserRegisterTest extends BaseTest {
+    @Test
+    public void testUserRegister() {
+        loginPage.clickRegisterLink();
+        sleep(2000);
```

Comment:
Yes, I am aware that the implicit wait is not a best practice and can slow down the execution of test scripts... 
I used it just to see physically how the test is executed the first couple of times. 

Since I should use the explicit wait whenever the loading of a particular element takes some time, will try to use the right combination of Expected Conditions that can be used in Explicit Wait.